docker-compose.yml

```diff
@@ -137,10 +137,6 @@ services:
       - HYPERSWITCH_SERVER_URL=${HYPERSWITCH_SERVER_URL:-http://hyperswitch-server:8080}
       - HYPERSWITCH_CLIENT_URL=${HYPERSWITCH_CLIENT_URL:-http://localhost:9050}
       - SELF_SERVER_URL=${SELF_SERVER_URL:-http://localhost:5252}
-      - SDK_ENV=${SDK_ENV:-local}
-      - ENV_SDK_URL=${ENV_SDK_URL:-http://localhost:9050/HyperLoader.js}
-      - ENV_BACKEND_URL=${ENV_BACKEND_URL:-http://localhost:8080}
-      - ENV_LOGGING_URL=${ENV_LOGGING_URL:-http://localhost:3100}
```

Comment:
can you add these variables back?

Reply:
Okay, will add it.

I'm getting this errors from morning on `hyperswitch-router:standalone`

```Unable to deserialize application configuration: connector_customer.payout_connector_list: Some errors occurred:
Unable to deserialize `stripe` as `api_models::enums::PayoutConnectors`: Matching variant not found
thread 'main' panicked at crates/router/src/bin/router.rs:14:10:
Unable to construct application configuration: ConfigurationError(Some errors occurred:
Unable to deserialize `stripe` as `api_models::enums::PayoutConnectors`: Matching variant not found)
note: run with `RUST_BACKTRACE=1` environment variable to display a backtrace```

@lsampras do you have any idea regarding this?

Reply:
Oh, this seems to be caused by recent changes to the `docker_compose.toml` file.

I would have suggested checking out to the `v1.107.0` tag before running the Docker Compose setup, but since you need to open a PR against the `main` branch, that isn't an option. You can remove `stripe` from the `payout_connector_list` line in the [`docker_compose.toml` file](https://github.com/juspay/hyperswitch/blob/b0133f33693575f2145d295eff78dd07b61efcda/config/docker_compose.toml#L398) to be able to run the Docker Compose setup, and you need not push the `docker_compose.toml` change as part of this PR.

Reply:
Thanks for the help @SanchithHegde 
Also when I'm trying to preview payment from control center UI is authenticating request with `beta.hyperswitch.io` is that expected?
Should I use key created there.